tasks/src/image-to-image/data.ts

```diff
@@ -0,0 +1,58 @@
+import { type TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Synthetic dataset, for image relighting",
+			id: "VIDIT"
+		},
+		{
+			description: "Multiple images of celebrities, used for facial expression translation"
+			id: "CelebA"
+		}
+	],
+	demo:     {
+		inputs: [
+			{
+				filename: "image-to-image-input.jpeg",
+				type:     "img",
+			},
+		],
+		outputs: [
+			{
+				filename: "image-to-image-output.png",
+				type:     "img",
+			},
+		]
+	},
+	isPlaceholder: false,
+	metrics:       [
+		{
+			description: "Peak Signal to Noise Ratio (PSNR) is an approximation of the human perception, considering the ratio of the absolute intensity with respect to the variations. Measured in dB, a high value indicates a high fidelity.",
+			id:          "PSNR"
+		},
+		{
+			description: "Structural Similarity Index (SSIM) is a perceptual metric which compares the luminance, constrast and structure of two images. The values of SSIM range between -1 and 1, and higher values indicate closer resemblance to the original image.",
+			id:          "SSIM"
+		},
+		{
+			description: "Inception Score (IS) is an analysis of the labels predicted by an image classification model when presented with a sample of the generated images.",
+			id:          "IS"
+		},
+	],
+	models:        [
```

Comment:
On top of these models, maybe we could add https://huggingface.co/lambdalabs/sd-image-variations-diffusers what do you think?